app/routines/search_exercises.rb

```diff
@@ -15,28 +15,18 @@ class SearchExercises
 
   protected
 
-  def exec(params = {})
+  def exec(params = {}, options = {})
     params[:ob] ||= [{number: :asc}, {version: :desc}]
-    run(:search, relation: Exercise.joins(:publication)
-                                   .preload(
-                                     :attachments,
-                                     :logic,
-                                     :tags,
-                                     publication: [:derivations,
-                                                   authors: :user,
-                                                   copyright_holders: :user,
-                                                   editors: :user],
-                                     questions: [
-                                       :hints,
-                                       answers: :stem_answers,
-                                       stems: [:stylings, :combo_choices]
-                                     ]
-                                   ),
+
+    latest_only = true
```

Comment:
Took me a sec to figure out what the `latest_only`s were doing.  Can you add a comment here along the lines of:

```ruby
# By default, only return the latest exercises.  If certain search conditions are specified,
# this "latest" condition may be disabled.
```